packages/video_player/video_player/test/video_player_test.dart

```diff
@@ -95,6 +95,22 @@ class _FakeClosedCaptionFile extends ClosedCaptionFile {
 }
 
 void main() {
+  void verifyAppLifeCycle(
```

Comment:
nit: this name isn't very helpful.  Something like verifyPlayingWhenAppLifeCyclePaused?